app/models/form/sales/questions/buyer1_age_known.rb

```diff
@@ -18,6 +18,9 @@ def initialize(id, hsh, page)
         {
           "age1_known" => 1,
         },
+        {
+          "age1_known" => 2,
```

Comment:
Does this need a matching definition in `config/forms/2022_2023.json`?

Reply:
Hmm, I'd say out of scope for this ticket, but merging this would make it easier to adjust the lettings logs ones as well if we need to

Reply:
Ah I hadn't clocked that `config/forms/2022_2023.json` was only relevant for lettings - I assumed it had stuff for both lettings and sales in it.